src/managementEnforcer.ts

```diff
@@ -395,8 +388,17 @@ export class ManagementEnforcer extends InternalEnforcer {
    *                    means not to match this field.
    * @return succeeds or not.
    */
-  public async removeFilteredNamedGroupingPolicy(ptype: string, fieldIndex: number, ...fieldValues: string[]): Promise<boolean> {
-    const ruleRemoved = await this.removeFilteredPolicyInternal('g', ptype, fieldIndex, fieldValues);
+  public async removeFilteredNamedGroupingPolicy(
```

Comment:
To many new lines here, please stop ``prettier`` doing this.

And please separate formatting code with functional change. Because it's hard for me to know where is changed.

Reply:
Fixed